package.json

```diff
@@ -15,17 +16,26 @@
     "test": "jest",
     "watch-test": "jest --watchAll",
     "example": "poi example/index.js",
-    "example:build": "rm -rf example/dist && poi build",
+    "example:build": "poi build",
     "example:deploy": "git subtree push --prefix example/dist origin gh-pages",
     "version": "npm run example:deploy"
   },
+  "jest": {
+    "verbose": true,
+    "testURL": "http://localhost/"
+  },
   "poi": {
     "entry": "example/index.js",
     "outDir": "example/dist",
     "publicPath": "/vue-tribute/"
   },
   "bili": {
-    "format": ["cjs", "umd", "umd-min", "es"],
+    "format": [
```

Comment:
I don't think these `package.json` changes should be in this PR.

Reply:
Sorry, this is my mistake, I will fix it soon.